src/components/App/Issue/commons/styles.js

```diff
@@ -93,6 +93,20 @@ NoChartText = styled.div`
     }
   }
 `,
+  WarningWrapper = styled.div`
+    position: absolute;
+    top: 0px;
+    right: 10px;
+    z-index: 5;
+    & > p {
+      padding: 3px 10px;
+      font-size: 15px;
+      margin: 0 auto;
+      background: #F4E4AB;
```

Comment:
See if `theme.js` has this color code already. If yes then use that else add this color in theme.js and use from there.